### DISCUSSION TITLE
vacation

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#1207] [2023 Attraction Contributions]
where do you go on your next vacation? any exotic destinations?

**Comment by MirkosMangle** [comment]
france and greece. Excited already

**Comment by hernessteve** [comment]
We haven’t yet decided on our next travel destination, but we’re leaning towards trying something completely different this time. Instead of the usual vacation spots, we’re exploring more unique options. At the moment, we're checking out [aferry](https://aferry.pissedconsumer.com/customer-service.html) to see what exciting ferry routes and destinations they offer. It feels like a great opportunity to experience something new, whether it’s a scenic coastal journey or discovering hidden gems along lesser-known routes. We're hoping AFerry can help us find the perfect adventure that’s off the beaten path.

**Comment by XelaAl** [comment]
Honestly, I’m still deciding! I’m torn between something relaxing like Bali and something more adventurous like Morocco. I just need sunshine, good food, and a break from reality. 
Where are you thinking of going?
